ocfweb/tv/templates/tv/desktop_usage.html

```diff
@@ -0,0 +1,16 @@
+<div class="page-tv-hours">
+  <p>There are <span id="desktop_num">_</span> desktops available.</p>
+  <script>
+    $(document).ready(function() {
+      function update() {
+        $.getJSON("{% url 'desktop_usage' %}", function(res) {
+          document.getElementById("desktop_num").innerHTML =
+          (res.total_desktops - res.desktops_in_use.length);
```

Comment:
Since you already include jQuery, I think this can be made simpler, something like `$('#desktop_num').text((res.total_desktops - res.desktops_in_use.length))`. Also I don't think you need to return the full list of desktops in use, but I'm guessing that's planned to be used in the future for a desktop map or something that shows which desktops are being used currently.

Reply:
+1 avoid setting innerhtml directly, it is easy to cause an xss vulnerability